build_runner/README.md

```diff
@@ -85,6 +85,9 @@ All the above commands support the following arguments:
   be provided.
 - `--[no-]fail-on-severe`: Whether to consider the build a failure on an error
   logged. By default this is false.
+- `--build-filter`: Build filters allow you to choose explicitly which files to
+  build instead of building entire directories. See further documentation on
+  this feature [here](../docs/partial_builds.md).
```

Comment:
Will this link work when the readme is on pub?

Reply:
Huh, good question. cc @jonasfj

Reply:
We might want to use an absolute URL here to be safe.

Reply:
Yeah ok I will just make it absolute, the relative url works better with IDEs (navigates to the actual file in your IDE), but the absolute uri still takes you to a good place.